src/ontology/cl-edit.owl

```diff
@@ -4157,7 +4158,6 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000066 "epitheliocyte")
 AnnotationAssertion(rdfs:label obo:CL_0000066 "epithelial cell")
 AnnotationAssertion(foaf:depiction obo:CL_0000066 "https://www.swissbiopics.org/api/image/Epithelial_cells.svg"^^xsd:anyURI)
 SubClassOf(obo:CL_0000066 obo:CL_0000000)
-SubClassOf(obo:CL_0000066 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000483))
```

Comment:
I don't think this is intended, please fix it since it is misclassifying many epithelial cells.